aws/requests.yaml

```diff
@@ -1,4 +1,11 @@
 releases:
+- name: "> 19.1.0"
```

Comment:
Unfortunately aws release 19.1.1 is already crafted and tested and we can't change it any more

```suggestion
- name: "> 19.1.1"
```

Reply:
All right, thanks!